edx_rest_framework_extensions/tests/test_permission.py

```diff
@@ -27,18 +25,60 @@ def test_has_permission(self, has_permission):
         """ Verify the method only returns True if the user is a superuser. """
         request = RequestFactory().get('/')
         request.user = factories.UserFactory(is_superuser=has_permission)
-        permission = IsSuperuser()
+        permission = permissions.IsSuperuser()
         self.assertEqual(permission.has_permission(request, None), has_permission)
 
     @ddt.data(None, AnonymousUser())
     def test_has_permission_with_invalid_users(self, user):
         """ Verify the method returns False if the request's user is not a real user. """
         request = RequestFactory().get('/')
         request.user = user
-        permission = IsSuperuser()
+        permission = permissions.IsSuperuser()
         self.assertFalse(permission.has_permission(request, None))
 
 
+@ddt.ddt
+class IsStaffTests(TestCase):
+    """ Tests for the IsStaff permission class. """
+
+    @ddt.data(True, False)
+    def test_has_permission(self, is_staff):
+        request = RequestFactory().get('/')
+        request.user = factories.UserFactory(is_staff=is_staff)
+        self.assertEqual(permissions.IsStaff().has_permission(request, None), is_staff)
+
+
+@ddt.ddt
+class IsUserInUrlTests(TestCase):
+    """ Tests for the IsUserInUrl permission class. """
+
+    @ddt.data(True, False)
+    def test_has_permission(self, has_user_in_url):
+        user = factories.UserFactory(username='foo')
+        url = '/'
+        if has_user_in_url:
+            url += '?username={}'.format(user.username)
```

Comment:
Do we need a test for a custom query param for username?

Reply:
See comment above.

Reply:
Please ignore. :)